components/DOBInput/index.jsx

```diff
@@ -23,7 +23,9 @@ const DOBInput = ({ className, preventMinor, isValidDate, viewDate, ...props })
   }
 
   return (
-    <DatePicker {...props} className={className} />
+    <DatePicker {...props}
+        className={className}
```

Comment:
Did you re-organize these or is that your IDE?

Reply:
I think that was my IDE I never opened up date picker